docs/TheBook/TechnologyTutorial.md

```diff
@@ -0,0 +1,797 @@
+Node1 is in the DMZ or on the IT side of your company. Node2 sends its information to Node1 and Node1 "forwards" that information to the Cloud.
+
+You can have any number of forwarder nodes to overcome any network topology a customer might have!
+You can also use redundant paths by adding a second Node to the IT (DMZ) and have Node2 point at two nodes simply by separating the ServiceRoutes with a semi-colon:
+```
+ServiceRoute = "ws://localhost:8700; ws://anotherip:anotherport;..."
+```
+
+![Img11](Images/img11.png)Showing a redundant path in the IT Relay and a custom Azure IoT Hub Plugin in the Cloud-Relay.
+
+
+You can also have multiple nodes on the OT floor connecting to Node2 and have it send Information (called "forwarding" or "relaying") on to the next node.
+
+As you can see you can create very complex mesh setups using C-DEngine nodes. 
+
+With this in mind you can setup any combination of nodes you can imaging - or better - fitting the needs of your customer topology.
+
+> As a rule of thumb always connect from a lower security network into a higher security network. NEVER the other way around! And IT devices/virtual machines/PCs/Servers are much more secure than OT machines or devices.
+
+To bring this scenario even further, you can now create another node in a completely different region of the world and connect to the Cloud-Relay. As long as they share the same ScopeID, data will flow to that node as well.
+
+## Chapter 10 - Pushing data into Azure IoT Hub
+
+A core scenario of the C-DEngine is the on-premise datacollection for Cloud Services such as the Azure IoT Hub. 
+
+***PLUGIN Azure IoT Hub***
+
+Elaborate!!
+
+## Chapter 11 - Securing the Host using SSL
+
+Depending on the host framework you are using (ASP.NET, IIS, Windows Service or Command Line) the procedure for securing your host is different. 
+Since the C-DEngine is based on standard HTTP/WebSockets code, adding SSL/TLS to your host is more administrative work than coding. In fact you do not have to write a single line of code to enable SSL/TLS.
+
+You can require only secure connections to a host by setting the MyStationPort and MyStationWSPort to 443. This will automatically require the connecting node to use HTTPS or WSS. 
+
+>The internal C-DEngine WebServer will only accept HTTP/WSS requests on Port 443. If you need/want to use a different port for incoming secure connections, you have to host the C-DEngine node inside IIS or ASP.NET.
+
+To learn more about how to setup your hosting PC check out this document: [https://github.com/TRUMPF-IoT/cdeDocs/blob/master/docs/Concepts/C-DEngineSecureOn-premiseRelays.pdf](https://github.com/TRUMPF-IoT/cdeDocs/blob/master/docs/Concepts/C-DEngineSecureOn-premiseRelays.pdf)
+
+There are many other security related configuration settings at the disposal of the Node Administrator and Developer.
+
+### Client Sided Certificates
```

Comment:
Just "Client Certificates", I think. Or "Client-side Certificates" if you want the side word in there?